subprojects/packagefiles/frozen/LICENSE.build

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
This should not be the license of the project, but for build file. I think we want to keep the same license for all wraps.

Reply:
I'm the past @jpakkane said they need to be the same as the project they're for, i think that's in the docs too

Reply:
Hum, could be worth clarifying this, I thought the intention was to keep them all under a permissive license. What if we have a wrap for a gpl project?

I was actually considering to set a single license file at the root of wraps, applying to the whole repository. Then release script can copy it in all patch tarballs automatically and we can stop duplicating it in packagefiles.

Reply:
The original reason for that was that we want to make it as easy as possible for projects to grab the wrapdb definitions and add them upstream.

Reply:
Replaced.